release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -132,6 +132,11 @@ You can now configure user-defined labels and tags in Google Cloud Platform (GCP
 
 In {product-title} 4.14, you can install a cluster on Microsoft Azure in a restricted network for installer-provisioned infrastructure (IPI) and user-provisioned infrastructure (UPI). For IPI, you can create an internal mirror of the installation release content on an existing Azure Virtual Network (VNet). For UPI, you can install a cluster on Microsoft Azure by using infrastructure that you provide. For more information, see xref:../installing/installing_azure/installing-restricted-networks-azure-installer-provisioned.adoc#installing-restricted-networks-azure-installer-provisioned[Installing a cluster on Azure in a restricted network] and xref:../installing/installing_azure/installing-restricted-networks-azure-user-provisioned.adoc#installing-restricted-networks-azure-user-provisioned[Installing a cluster on Azure in a restricted network with user-provisioned infrastructure].
 
+[id="ocp-4-14-bare-metal-installation-disk-id"]
+==== Specifying installation disks using by-path device alias
```

Comment:
Maybe one of these?
```suggestion
==== Specifying installation disks using by-path device aliases
```
or
```suggestion
==== Specifying installation disks using a by-path device alias
```

Reply:
Thanks, I went with "a by-path device alias".